### PR TITLE
New package: iMobieInc.AnyUnlock version 2.0.1.1

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/118751)

### CODE REVIEW

**Comment by stephengillie** [comment]
Should this be a portable package?

File: manifests/i/iMobieInc/AnyUnlock/2.0.1.1/iMobieInc.AnyUnlock.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: iMobieInc.AnyUnlock
+PackageVersion: 2.0.1.1
+Installers:
+- InstallerUrl: https://www.imobie.com/go/download.php?product=auo&id=2
```
Comment (Trenly) [inline manifests/i/iMobieInc/AnyUnlock/2.0.1.1/iMobieInc.AnyUnlock.installer.yaml:7]:
```suggestion
- InstallerUrl: https://dl.imobie.com/anyunlock-iphone-password-unlocker-en-official-setup.exe
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
This package doesn't seem to modify the system. Is that expected?

**Comment by Trenly** [comment]
Stale PR; Please update or close
[Policy] Needs-Author-Feedback

**Pull request closed without merging**